scripts/run_backend_tests.sh

```diff
@@ -85,13 +85,14 @@ for arg in "$@"; do
   fi
 done
 
+$PYTHON_CMD scripts/build.py
+
 # Compile typescript files
 echo "Compiling typescript..."
 $NODE_MODULE_DIR/typescript/bin/tsc --project .
 
 echo "Compiling webpack..."
-$NODE_MODULE_DIR/webpack/bin/webpack.js --config webpack.prod.config.ts
-$PYTHON_CMD scripts/build.py
```

Comment:
Just wondering -- the changes look fine, but what was the rationale behind moving build.py earlier? I do think it's fine this way, just curious.